database/opensource/opensource_blogs.json

```diff
@@ -5,5 +5,19 @@
       "url": "https://chrissycodes.hashnode.dev/",
       "category": "opensource",
       "subcategory": "opensource_blogs"
+   },
+   {
+      "name": "Ways to get more contributors to your open source project",
+      "description": "Several ways to get more contributors to your Open Source project.",
```

Comment:
[Proofreading] Thank you so much for choosing my blog! :) Unfortunately, I already added it to the[blog section](https://linkshub.vercel.app/opensource_blogs), so consider replacing it with another woman who writes about open source. 
```suggestion
      "url": "https://chrissycodes.hashnode.dev/",
      "category": "opensource",
      "subcategory": "opensource_blogs"
   },
   {
      "name": "Ways to get more contributors to your open source project",
      "description": "Several ways to get more contributors to your Open Source project.",
```